vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,214 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gmps: <http://linked.data.gov.au/def/qld-resource-permit-status/> .
+
+<http://linked.data.gov.au/def/qld-resource-permit-status> a owl:Ontology .
+
+gmps:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-09-13"^^xsd:date ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:source "https://myminesonline.business.qld.gov.au" ;
+    skos:definition "The stage in the life cycle of a Queensland Resource Permit."@en ;
+    skos:hasTopConcept gmps:application,
+        gmps:discarded,
+        gmps:granted,
+        gmps:non-current,
+        gmps:not-accepted,
+        gmps:prelodgement,
+        gmps:submitted ;
+    skos:prefLabel "Queensland Resource Permit Status"@en .
+
+gmps:abandoned a skos:Concept ;
+    skos:definition "The Resource Permit was not granted and has been abandoned by the applicant."@en ;
+    skos:broader gmps:application,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Abandoned"@en .
+
+gmps:accepted a skos:Concept ;
+    skos:definition "The application has been accepted."@en ;
+    skos:broader gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Accepted"@en .
+
+gmps:awaiting-payment-confirmation a skos:Concept ;
+    skos:definition "Payment has been made, but the Department needs confirmation of this."@en ;
+    skos:broader gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Awaiting payment confirmation"@en .
+
+gmps:competing-application a skos:Concept ;
+    skos:definition "This application represents a competing application."@en ;
+    skos:broader gmps:application,
+        gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Competing application"@en .
+
+gmps:conditionally-surrendered a skos:Concept ;
+    skos:definition "The application has been conditionally surrendered."@en ;
+    skos:broader gmps:application,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Conditionally surrendered"@en .
+
+gmps:conversion-of-ml-to-mc a skos:Concept ;
+    skos:definition "The Mining Lease has been converted to a Mining Claim."@en ;
+    skos:broader gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Conversion of ML to MC"@en .
+
+gmps:expired a skos:Concept ;
+    skos:definition "The Resource Permit has expired."@en ;
+    skos:broader gmps:granted,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Expired"@en .
+
+gmps:finalised a skos:Concept ;
+    skos:definition "The Resource Permit has no outstanding requirements."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Finalised"@en .
```

Comment:
How are expired and finalised different? Can you please ask what the requirements are on an expired permit that distinguish it from finalised?

Reply:
Any explanation on this? I'm still assuming/guessing the difference without definitions that make distinction clear.